[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.VLobYbY4mC"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:35 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:48 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:56:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:13 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:28 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:42 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"jquery-cdn-google-api\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.pL7JRlA643"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:35 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:48 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:56:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:13 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:28 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:42 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"jquery-cdn-google-api\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.qjKEEdNrU5"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:10 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/admin/jpmeta.class.php(18): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(62): jpmeta->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(31): journalpress->canpost()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->canpost, jpmeta->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:54:20 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_jp_journals WHERE journalUse != :param0  ORDER BY journalUse, journalUser, journalServer | parameters: no</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_jp_journals.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/journalpress/lib/jpposts.class.php(23): wpdb->get_results('SELECT * FROM w...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/journalpress/journalpress.php(72): jpposts->__construct()\n#4 /wordpress/wp-content/plugins/journalpress/journalpress.php(33): journalpress->catchposts()\n#5 /wordpress/wp-content/plugins/journalpress/journalpress.php(102): journalpress->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('journalpress/jo...')\n#8 /internal/eval.php(20): activate_plugin('journalpress/jo...')\n#9 {main}</pre>\n for query SELECT * FROM wp_jp_journals WHERE journalUse != 'no' ORDER BY journalUse, journalUser, journalServer made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/journalpress/journalpress.php'), journalpress->__construct, journalpress->catchposts, jpposts->__construct, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:35 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:55:48 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:56:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_generate_attachment_metadata() in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php:4224\nStack trace:\n#0 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(72): mycred_upload_default_point_image()\n#1 /wordpress/wp-content/plugins/mycred/includes/mycred-setup.php(33): myCRED_Setup->setup_default_point_type()\n#2 /wordpress/wp-content/plugins/mycred/includes/mycred-install.php(94): myCRED_Setup->load()\n#3 /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php(3510): myCRED_Install::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mycred_plugin_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_mycred...', false)\n#8 /internal/eval.php(20): activate_plugin('mycred/mycred.p...')\n#9 {main}\n  thrown in /wordpress/wp-content/plugins/mycred/includes/mycred-functions.php on line 4224",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:13 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:28 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  },
  {
    "message": "Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "jquery-cdn-google-api",
    "details": "[30-May-2025 01:57:42 UTC] PHP Deprecated:  Required parameter $v_id follows optional parameter $p_name in /wordpress/wp-content/plugins/myworks-sync-for-xero/includes/class-lib.php on line 939\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"jquery-cdn-google-api\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/jquery-cdn-google-api.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/j/jquery-cdn-google-api/error.json"
  }
]
